datasets/afvalwijzer/dataset.json

```
@@ -172,7 +172,15 @@
           },
           "afvalwijzerGeometrie": {
             "$ref": "https://geojson.org/schema/Point.json",
-            "description": "Geometrie van het type POINT van het adres in RD stelsel (epsg:28992)"
+            "description": "Geometrie van het type POINT van het adres in RD stelsel (EPSG:28992)"
+          },
+          "afvalwijzerWgs84Lon": {
+            "type": "number",
+            "description": "Geometrie van het type POINT van het adres in WGS84 (EPSG:4326)"
```

De wijziging van de beschrijvingen is niet doorgekomen:

```suggestion
            "description": "Westerlengte van het adres in WGS84 (EPSG:4326)"
```